index.html

```diff
@@ -5104,9 +5106,10 @@ <h2>IANA Considerations</h2>
 Extensions of Verifiable Credentials Data Model 1.0</a>
         </li>
       </ul>
+</section>
 
 <section id="vc-ld-media-type">
-  <h2>application/vc+ld+json</h2>
+  <h2>Media Type Registration: `application/vc+ld+json`</h2>
```

Comment:
```suggestion
  <h2>application/vc+ld+json</h2>
```

Reply:
https://github.com/w3c/json-ld-syntax/pull/415/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R13429

Reply:
@OR13 — 
1. Why delete `Media Type Registration:`?
2. What is the relevant piece of that GIANT diff to which you linked?

Saving yourself writing time, as here, radically increases the time your readers must devote to deciphering your meaning. Please don't do this.